### PR TITLE
Basic build instructions

### PR DESCRIPTION
**Pull request by itscynxx**
Tried running the instructions found from the GitHub workflow and that didn't work, asked in the Discord and got this which did work

Maybe wording could be better though?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # NorthstarDiscordRPC
 
 Plugin to integrate Northstar with Discord's SDK to enable [rich presence](https://discord.com/rich-presence).
+
+# Building
+
+The Northstar RPC plugin uses [cargo](https://doc.rust-lang.org/cargo/index.html) to download dependencies and build code
+
+Run `cargo build --release` in the root directory of the plugin. The output will be found in `target/release`.
```
Comment (catornot) [inline README.md:9]: release is not required to build the plugin, it only disables the plugin from throwing errors and forces it to shutdown instead also ofc it optimizes it.

**Review by catornot — CHANGES_REQUESTED** [review]
Still don't really know if this is needed

File: README.md (hunk 1 above)
Comment (GeckoEidechse) [reply README.md:9]: I though @itscynxx had issues with getting the plugin running without compiling as release? Or did I confuse something here? ^^"

Comment (catornot) [reply README.md:9]: `cargo build` should work fine. idk why someone would have issues unless they are building on linux :|

**Comment by GeckoEidechse** [comment]
Imma move this to post plugins v3

File: README.md (hunk 1 above)
Comment (GeckoEidechse) [reply README.md:9]: So running `cargo build` worked just fine for me on Windows (on plugins v3).

**Comment by itscynxx** [comment]
Closing as isn't needed

**Pull request closed without merging**